netket/hilbert/doubled_hilbert.py

```diff
@@ -122,29 +123,23 @@ def _numbers_to_states(self, numbers, out):
         # 2 -> [0,0,1,0]
         # etc...
 
-        n = self.physical.size
         dim = self.physical.n_states
-        left, right = np.divmod(numbers, dim)
-
-        self.physical.numbers_to_states(left, out=out[:, 0:n])
-        self.physical.numbers_to_states(right, out=out[:, n : 2 * n])
+        left, right = jnp.divmod(numbers, dim)
```

Comment:
For now? Or is there a reason?
```suggestion
        left, right = np.divmod(numbers, dim)
```